package/PartSeg/common_gui/napari_image_view.py

```diff
@@ -563,3 +561,12 @@ def reset_image_size(self):
 
     def set_theme(self, theme: str):
         self.viewer.theme = theme
+
+
+class NapariQtViewer(QtViewer):
+    def dragEnterEvent(self, event):  # pylint disable=no-self-use
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/Czaki/PartSeg/pullRequest?prid=5798467)